medusa/classes.py

```diff
@@ -15,7 +15,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Medusa. If not, see <http://www.gnu.org/licenses/>.
-
+"""Classes module."""
```

Comment:
These kind of module docstrings are useless.  I'd rather have a flake error than useless documentation, that at least serves to show where improved documentation is needed.  Give some useful information.  For example what kind of classes?  Math? Ballet? :D